src/components/FloatingDropdown/FloatingDropdown.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
+import { isUniEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
+import { isUniTestkitExists } from 'wix-ui-test-utils/vanilla';
+import { mount } from 'enzyme';
+import { TPAComponentsWrapper } from '../../test/utils';
+import { floatingDropdownDriverFactory } from './FloatingDropdown.driver';
+import { FloatingDropdown } from './';
+import { floatingDropdownTestkitFactory } from '../../testkit';
+import { floatingDropdownTestkitFactory as enzymeFloatingDropdownTestkitFactory } from '../../testkit/enzyme';
+import { FloatingDropdownProps } from './FloatingDropdown';
+import { getFloatingDropdownTestProps } from './test-props';
+
+describe('FloatingDropdown', () => {
+  const createDriver = createUniDriverFactory(floatingDropdownDriverFactory);
+
+  const testProps = getFloatingDropdownTestProps();
+
+  it('should render', async () => {
+    const driver = createDriver(<FloatingDropdown {...testProps} />);
+    expect(await driver.exists()).toBe(true);
+  });
+
+  describe('testkit', () => {
+    it('should exist', async () => {
+      expect(
+        await isUniTestkitExists(
+          <FloatingDropdown {...testProps} />,
+          floatingDropdownTestkitFactory,
+          {
+            dataHookPropName: 'data-hook',
+          },
+        ),
+      ).toBe(true);
+    });
+  });
+
```

Comment:
Do we have any logic we need to test here 🤔 ?

Reply:
I think, it makes sense to add some tests for:

1. checking if dropdown appears after a click on enabled FloatingDropdown component 
2. does not appear for disabled
3. making sore, that selected options changes in dropdown base after selecting it.

I will add these.

Reply:
@jonathanadler added these: https://github.com/wix/wix-ui-tpa/pull/313/commits/a40eef63ae3d9fca60631f5dced87ae71be3857f